### DISCUSSION TITLE
Error when a parameter is added in the Resource constructor

### DISCUSSION THREAD

**Discussion by felipe-dsdev** [laravel/nova-issues#4655] [Ideas & Feature Requests]
- Laravel Version: 9.21.3
- Nova Version: 4.12.12
- PHP Version: 8.1.1
- Database Driver & Version: MariaDB 10.5.12
- Operating System and Version: Amazon Linux 2 (Lambda Function with bref php-81-fpm layer)
- Browser type and version: Chrome 103.0.5060.134 64 bits
- Reproduction Repository: Fresh Nova Installation

### Description:
I'm extending the Laravel\Nova\Resource class, however when adding additional parameters in the constructor for the Laravel DI to include the classes, the following error is returned:

`Too few arguments to function Laravel\Nova\Tools\ResourceManager::Laravel\Nova\Tools\{closure}(), 1 passed and exactly 2 expected`

![image](https://user-images.githubusercontent.com/101280418/182298452-990aefd1-65ab-4e44-9110-a2f7c631c96e.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:

- Create a new resource (like in this command: `php artisan nova:resource Post`)
- Add constructor method with a class as required parameter
- Access Laravel Nova in Browser and the error will be displayed

**Comment by crynobone** [comment]
Such customization isn't supported by Nova at the moment.
